package.json

```
@@ -40,30 +40,30 @@
     "lint": "eslint src/**/*.ts"
   },
   "peerDependencies": {
-    "@mikro-orm/core": "^5.0.0 || ^6.0.0-dev.0",
-    "@nestjs/common": "^8.0.0 || ^9.0.0 || ^10.0.0",
-    "@nestjs/core": "^8.0.0 || ^9.0.0 || ^10.0.0"
+    "@mikro-orm/core": "^5.0.0 || ^6.0.0",
```

your changes are already breaking for v5, lets remove it from the peer deps so we dont have to support both things at the same time
```suggestion
    "@mikro-orm/core": "^6.0.0",
```

---

I missed it! Thank you.